interface/src/app/gene/network/network.js

```diff
@@ -113,6 +113,7 @@ angular.module('adage.gene.network', [
           .geneText(function(d) {
             return d.label;
           })
+          .bgCorrection(false)    // disable background correction
```

Comment:
I forgot to mention that the `bgCorrection` variable was introduced into d3.network.js after I discussed the algorithm that Aaron used to rank the genes (to decide which genes should be displayed in the network with slider that controls the max # of genes). Here is how it was implemented in `d3.network.js`:

* On line 42 of `d3.network.js`, you can see that `bgCorrection` is set to `true` by default.;

* Lines 493-499 defines an API function that allows user to reset `bgCorrection` variable;

* Lines 106-111 are using two different algorithms to rank the genes based on whether `bgCorrection` is set or not.

That is why in adage-server, we are calling `bgCorrection(false)` to set this flag to `false` explicitly. Does this make sense?

Reply:
Well, I followed the code, but I don't understand why we have to alter the ranking algorithm. Could you include a comment giving the rough idea behind each algorithm (enough to understand why we might use one vs. the other) at lines 106-111? I think the comment at 104-105 needs to be updated and expanded for each implementation because it doesn't apply to both cases.

Reply:
The ranking algorithms are lines 66-112. I don't think I can explain better than that. It basically considers both the edge weight and number of edges when calculating the rank. The key difference between the two algorithms is on lines 107 and 110. 

The comments on lines 104-105 were written by Aaron. I prefer modifying them later, not in this PR.

I have created an issue to keep track it. Need to figure out the best way to explain it to a regular user.

Reply:
Tracking as an issue works for me. I think it will be confusing, and right now I could not explain why it does what it does what it does if a user asked.